src/components/Initiatives/InitiativeDetailsPage/InitiativeMetadataCard/InitiativeMetadataCard.tsx

```diff
@@ -70,6 +110,26 @@ export const InitiativeMetadataCard = ({
             <MarkdownContent content={initiative.description} />
           </Box>
         )}
+        <Box className={classes.markdownBox}>
+          <CaptionTypography variant="caption">
+            Notification Schedule
+          </CaptionTypography>
+          <MarkdownContent content={getNotificationDisplayText(initiative)} />
+          {!hideLink && (
```

Comment:
I forget: do we have any sense of the user's role/permissions in Cortex here? Because we ideally don't show this link to users who can't edit the permissions.

Reply:
Apparently we load them but don't ever use them? Also backstage runs from an API Key so don't know how it would even be per user?

Reply:
We load and it always errors out, not going to worry about it for now.